### PR TITLE
Scripts to check issue 1884

### PR DESCRIPTION
**Pull request by HSavynetska**
ATF Test Scripts to check [#1884](https://github.com/smartdevicelink/sdl_core/issues/1884)

This PR is **not ready** for review.
This is **draft** version of the script.

### Summary
A test script to verify that issue is fixed

### ATF version
[develop](https://github.com/smartdevicelink/sdl_atf/tree/develop)

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Defects/5_0/1884_SDL_behavior_in_case_HMI_responds_with_WARNINGS_to_at_least_one_VR_DeleteCommands_AND_app_sends_PerfromInteraction_related_to_this_ChoiceSet.lua [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('user_modules/sequences/actions')
+local runner = require('user_modules/script_runner')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local grammarIDValue
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1884_SDL_behavior_in_case_HMI_responds_with_WARNINGS_to_at_least_one_VR_DeleteCommands_AND_app_sends_PerfromInteraction_related_to_this_ChoiceSet.lua]: value is not used

File: test_scripts/Defects/5_0/1884_SDL_behavior_in_case_HMI_responds_with_WARNINGS_to_at_least_one_VR_DeleteCommands_AND_app_sends_PerfromInteraction_related_to_this_ChoiceSet.lua [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+local deleteAllParams = {
+  requestParams = deleteRequestParams,
+  responseVrParams = deleteResponseVrParams
+}
+
+local performParams = {
+  initialText = "TextInitial",
+  interactionMode = "MANUAL_ONLY",
+  interactionChoiceSetIDList = { 100 },
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1884_SDL_behavior_in_case_HMI_responds_with_WARNINGS_to_at_least_one_VR_DeleteCommands_AND_app_sends_PerfromInteraction_related_to_this_ChoiceSet.lua]: id of created choiceSet is 1, not 100

File: test_scripts/Defects/5_0/1884_SDL_behavior_in_case_HMI_responds_with_WARNINGS_to_at_least_one_VR_DeleteCommands_AND_app_sends_PerfromInteraction_related_to_this_ChoiceSet.lua [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    { type = "TEXT", text = "pathToFile1" }
+  }
+}
+
+--[[ Local Functions ]]
+local function createInteractionChoiceSet()
+  local corId = common.getMobileSession():SendRPC("CreateInteractionChoiceSet", requestchoiceParams)
+  common.getHMIConnection():ExpectRequest("VR.AddCommand",
+    { cmdID = 111, type = "Choice", vrCommands = { "Choice111" } })
+  :Times(1)
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1884_SDL_behavior_in_case_HMI_responds_with_WARNINGS_to_at_least_one_VR_DeleteCommands_AND_app_sends_PerfromInteraction_related_to_this_ChoiceSet.lua]: times(1) is default value, so can be removed

File: test_scripts/Defects/5_0/1884_SDL_behavior_in_case_HMI_responds_with_WARNINGS_to_at_least_one_VR_DeleteCommands_AND_app_sends_PerfromInteraction_related_to_this_ChoiceSet.lua [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+  :Do(function(_, data)
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+  common.getMobileSession():ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
+end
+
+local function deleteInteractionChoiceSet(params)
+	local cid = common.getMobileSession():SendRPC("DeleteInteractionChoiceSet", params.requestParams)
+
+	params.responseVrParams.appID = common.getHMIAppId()
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1884_SDL_behavior_in_case_HMI_responds_with_WARNINGS_to_at_least_one_VR_DeleteCommands_AND_app_sends_PerfromInteraction_related_to_this_ChoiceSet.lua]: is not used

File: test_scripts/Defects/5_0/1884_SDL_behavior_in_case_HMI_responds_with_WARNINGS_to_at_least_one_VR_DeleteCommands_AND_app_sends_PerfromInteraction_related_to_this_ChoiceSet.lua [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+	EXPECT_HMICALL("VR.DeleteCommand", { cmdID = 111, type = "Choice" })
+	:Do(function(_,data)
+    grammarIDValue = data.params.grammarID
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1884_SDL_behavior_in_case_HMI_responds_with_WARNINGS_to_at_least_one_VR_DeleteCommands_AND_app_sends_PerfromInteraction_related_to_this_ChoiceSet.lua]: is not used

File: test_scripts/Defects/5_0/1884_SDL_behavior_in_case_HMI_responds_with_WARNINGS_to_at_least_one_VR_DeleteCommands_AND_app_sends_PerfromInteraction_related_to_this_ChoiceSet.lua [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+		common.getHMIConnection():SendResponse(data.id, data.method, "WARNINGS", {})
+	end)
+
+	common.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS"})
+	common.getMobileSession():ExpectNotification("OnHashChange")
+end
+
+local function sendPerformInteraction_SUCCESS()
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1884_SDL_behavior_in_case_HMI_responds_with_WARNINGS_to_at_least_one_VR_DeleteCommands_AND_app_sends_PerfromInteraction_related_to_this_ChoiceSet.lua]: resultCode is not SUCCESS

File: test_scripts/Defects/5_0/1884_SDL_behavior_in_case_HMI_responds_with_WARNINGS_to_at_least_one_VR_DeleteCommands_AND_app_sends_PerfromInteraction_related_to_this_ChoiceSet.lua [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("Register App", common.registerApp)
+runner.Step("Activate App", common.activateApp)
+
+runner.Title("Test")
+runner.Step("CreateInteractionChoiceSet", createInteractionChoiceSet)
+runner.Step("DeleteInteractionChoiceSet", deleteInteractionChoiceSet, {deleteAllParams})
+runner.Step("Send PerformInteraction SUCCESS response", sendPerformInteraction_SUCCESS)
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/5_0/1884_SDL_behavior_in_case_HMI_responds_with_WARNINGS_to_at_least_one_VR_DeleteCommands_AND_app_sends_PerfromInteraction_related_to_this_ChoiceSet.lua]: resultCode is not success

**Comment by dboltovskyi** [comment]
Livio team please do not review this PR for now since it hasn't passed our internal review yet.  
Review and appropriate updates going be done till 1st of November.
Thank you.

**Comment by dboltovskyi** [comment]
Closed, since issue is not actual.

**Pull request closed without merging**